ts/Series/PackedBubble/PackedBubbleDataLabelOptions.d.ts

```diff
@@ -47,6 +51,7 @@ export interface PackedBubbleDataLabelOptions extends DataLabelOptions {
     );
     parentNodeTextPath?: DataLabelTextPathOptions;
     textPath?: DataLabelTextPathOptions;
+    animation?: PackedBubbleDataLabelAnimationObject;
```

Comment:
You're right - I applied this interface because `animation.defer` for `PackedBubble` was in the docs and I had to define an interface for `animation` because of a TypeScript issue that arised (`NetworkgraphSeries` is not assignable to `BubbleSeries`) because I used an additional `fadeDuration` property. 

Now, I've removed this property (as we'll use `style.transition`) so I could also get rid of this interface - the `PackedBubble` series is almost untouched.

However, the `animation.defer` didn't work & doesn't seem to be working now (for the same reason it did not work for `Networkgraph`). Should I implement a similar logic for the `PackedBubble` as I did for `Networkgraph`?

Reply:
Yes please, those two series types should work in the same way.